docs/archapi.md

```diff
@@ -398,6 +398,10 @@ actual penalty used is a multiple of this value (i.e. a weighted version of this
 
 Convert an `delay_t` to an actual real-world delay in nanoseconds.
 
+### DelayInfo getDelayFromNS(float v) const
+
+Convert a real-world delay in nanoseconds to a DelayInfo with equal min/max rising/falling values.
```

Comment:
Why not simply return a delay_t?

Reply:
We use DelayInfo in clock constraints, etc - but architectures currently have no API to create one. The alternative would be to use something other than DelayInfo in all those cases.

Reply:
Ah, I see, that makes sense. Maybe we should add some std constructors to DelayInfo then. (But no need to delay this PR for that imo.)